.github/workflows/ci.yaml

```diff
@@ -33,16 +33,30 @@ jobs:
       RAILS_ENV: test
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - name: Checkout Repository
```

Comment:
I think names like that are not necessary since the action name is pretty explanatory.